src/mc-patchnotes.html

```diff
@@ -11,6 +11,7 @@
                 <h1>Versions</h1>
             </nav>
             <main>
+                <noscript>You need to enable JavaScript for this website in order to function.</noscript>
```

Comment:
Maybe just "This page requires JavaScript, to retrieve the Minecraft launcher's patch notes. Please enable it and reload the page to continue."

Reply:
sure!